### DISCUSSION TITLE
screenshot method from html2image 2.0.1 no longer working after 2.0.2 released?

### DISCUSSION THREAD

**Discussion by jrb-github** [vgalin/html2image#107] [Q&A] [answered]
It doesn't seem to make a lot of sense, but recently I noticed an script I wrote based on html2image 2.0.1 that runs daily, fails to save a file to disk when calling the screenshot method.  The method returns a path suggesting that it did create a file, but no file is written to disk.  As I looked to see when this problem started, it was very close to when html2image 2.0.2 was released.  However, I am pinned to 2.0.1, so this doesn't seem to be the issue directly.  Further, if I install 2.0.2, my script fails with other errors I still need to debug.  My best guess is that some change to the connectivity to the browsers involved was known to be breaking soon, and that motivated the updates in 2.0.2, and I need to update?  Is this documented somewhere that I just couldn't find?

**Comment by vgalin** [comment, marked as answer]
Hi, the 2.0.2 version only introduces a small change:
> Default background color flag on Chrome/Chromium changed from '0' to '000000' due to unsupported value in Chromium version >= 1.1.1.

However the 1.1.1 version also introduces bugs that are breaking screenshots. Some people in #103 have chosen to temporarily downgrade their Chrome/Chromium version for the moment.
